src/main/java/com/palantir/gradle/revapi/config/RevapiConfig.java

```diff
@@ -41,14 +47,43 @@ public final RevapiConfig addVersionOverride(GroupNameVersion groupNameVersion,
                 .build();
     }
 
+    public final Set<AcceptedBreak> acceptedBreaksFor(GroupNameVersion groupNameVersion) {
+        return Streams.stream(Optional.ofNullable(acceptedBreaks().get(groupNameVersion.version())))
+                .flatMap(versionedAcceptedBreaks -> versionedAcceptedBreaks.acceptedBreaks()
+                        .get(groupNameVersion.groupAndName())
```

Comment:
this could be null, projects might not exist at the previous version

Reply:
Ooof, the horrors of `Multimaps` not deserializing properly in immutables strikes again (if I'd managed to get the `Multimap` version to work this would have been fine).